### DISCUSSION TITLE
WJHG / WECP - CBS | (US)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#476] [Show and tell]
### Channel ID

WJHGDT3.us

### Stream URL

https://playlistserver.aws.syncbak.com/playlist/13353734/master.m3u8?access_token=eyJhbGciOiJIUzI1NiIsImtpZCI6IkdyYXkyMDE2MDgyOSIsInN1YiI6IioiLCJ0eXAiOiJKV1QifQ.eyJleHAiOjE3OTA2MzQ5ODIsImlzcyI6IldMUyIsIm1kNSI6ImMzYWQ0YTdmMmUyYmFiMzg4NjVlYTZmMzkzMzZmYjcxIiwibWlkIjoiMTMzNTM3MzQifQ.pBrv9Z2h1cfbHlkWL_nETTG4BBhH0xgIrj9E4JYs568

### Notes

I Don't Know If The Signal Is From WJHG Or WECP But The The m3u8 Came From Here
https://www.wjhg.com/pf/api/v3/content/fetch/site-service?query=%7B%22section%22:%22%2Fvideo%22,%22websiteOverride%22:%22wjhg%22%7D&_website=wjhg

Livestream1:https://dai2-playlistserver.aws.syncbak.com/playlist/899080/master.m3u8?access_token=eyJhbGciOiJIUzI1NiIsImtpZCI6IkdyYXkyMDE2MDgyOSIsInN1YiI6IioiLCJ0eXAiOiJKV1QifQ.eyJleHAiOjE3OTAzNjg5ODAsImlzcyI6IldMUyIsIm1kNSI6IjdiYTY0ZTE5MzczOWE4Y2Y4YzU0MWRjOWM3OGUzMTliIiwibWlkIjoiODk5MDgwIn0.Jrb2-pvJfN0b-4VxltBaiIcSdA-Bfzf9SD6pV75MB7g

Livestream2:https://playlistserver.aws.syncbak.com/playlist/13353734/master.m3u8?access_token=eyJhbGciOiJIUzI1NiIsImtpZCI6IkdyYXkyMDE2MDgyOSIsInN1YiI6IioiLCJ0eXAiOiJKV1QifQ.eyJleHAiOjE3OTA2MzQ5ODIsImlzcyI6IldMUyIsIm1kNSI6ImMzYWQ0YTdmMmUyYmFiMzg4NjVlYTZmMzkzMzZmYjcxIiwibWlkIjoiMTMzNTM3MzQifQ.pBrv9Z2h1cfbHlkWL_nETTG4BBhH0xgIrj9E4JYs568

Livestream3:https://dai2-playlistserver.aws.syncbak.com/playlist/13354439/master.m3u8?access_token=eyJhbGciOiJIUzI1NiIsImtpZCI6IkdyYXkyMDE2MDgyOSIsInN1YiI6IioiLCJ0eXAiOiJKV1QifQ.eyJleHAiOjE3OTEzMTA0MTUsImlzcyI6IldMUyIsIm1kNSI6IjMzYjk3MTRkMjhmMjE2NzVlZjgxYzE4ODRmZDc4NmU5IiwibWlkIjoiMTMzNTQ0MzkifQ.PvuQWv-NMiR4ZYe57_px7UCN0NNyl8x6h8JCKSwZUUg

Livestream5:https://dai2-playlistserver.aws.syncbak.com/playlist/18625525/master.m3u8?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IkdyYXkyMDE2MDgyOSJ9.eyJtaWQiOiIxODYyNTUyNSIsImNpZCI6MTc0MjcsInNpZCI6MTU4LCJtZDUiOiI3MDE0OWI3OWMyOTI0MjY5MTQwYmY0MDE0ZDIyMmYyZiIsImlhdCI6MTYxMDQ3NTM1MywiaXNzIjoiU3luY2JhayAoVEcpIn0.fCdAiZnixSUkRT87xgjvXCRVpGGcoD5SLJkENFlEpSo

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by kaikyoofers** [comment]
Tokened

**Comment by AntiPontifex** [comment]
Dead issue: please close.

**Comment by freearhey** [comment]
https://playlistserver.aws.syncbak.com/playlist/13353734/master.m3u8?access_token=eyJhbGciOiJIUzI1NiIsImtpZCI6IkdyYXkyMDE2MDgyOSIsInN1YiI6IioiLCJ0eXAiOiJKV1QifQ.eyJleHAiOjE3OTA2MzQ5ODIsImlzcyI6IldMUyIsIm1kNSI6ImMzYWQ0YTdmMmUyYmFiMzg4NjVlYTZmMzkzMzZmYjcxIiwibWlkIjoiMTMzNTM3MzQifQ.pBrv9Z2h1cfbHlkWL_nETTG4BBhH0xgIrj9E4JYs568 (offline)
